### PR TITLE
Flex 2923 convert register device fitnesse tests to cucumber

### PR DESCRIPTION
**Pull request by BlietZ**
+ Network address changed to IpAddress

### CODE REVIEW

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/glue/steps/mocks/OslpDeviceSteps.java [hunk 1]
```diff
@@ -75,6 +78,10 @@
  */
 public class OslpDeviceSteps {
 
+		
+	@Autowired
```
Comment (MartijnSipsCGI) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/glue/steps/mocks/OslpDeviceSteps.java]: Fix your indentations. See http://documentation.opensmartgridplatform.org/Opensourcecommunity/ToolsguidelinesCI.html.

File: cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/glue/steps/mocks/OslpDeviceSteps.java [hunk 2]
```diff
@@ -839,7 +846,7 @@ public void theOSLPEventNotificationResponseContains(final String protocol,
     @Given("^the device sends a register device request to the platform over \"([^\"]*)\"$")
     public void theDeviceSendsARegisterDeviceRequestToThePlatform(final String protocol,
             final Map<String, String> settings) throws IOException, DeviceSimulatorException {
-
+    	try{
```
Comment (MartijnSipsCGI) [inline cucumber-tests-platform/src/test/java/com/alliander/osgp/cucumber/platform/glue/steps/mocks/OslpDeviceSteps.java]: The indentation is not correct.

**Comment by SamVanDijk** [comment]
test this please

**Comment by SamVanDijk** [comment]
retest this please

**Comment by BlietZ** [comment]
retest this please

**Comment by SamVanDijk** [comment]
retest this please

**Comment by BlietZ** [comment]
retest this please

**Comment by SamVanDijk** [comment]
retest this please

**Comment by SamVanDijk** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Comment by MartijnSipsCGI** [comment]
retest this please

**Pull request merged**